contrib/kafka/samples/kafka-event-source.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 The Knative Authors
```

Comment:
`kafka-sample-source.yaml` ?

Reply:
Same with above.  Addressing this by dropping the additional file and just giving a concrete example to use in tandem with the (existing) sample-source.yaml